### PR TITLE
Config for session lifetime

### PR DESCRIPTION
**Pull request by iceaxeliehne**

### CODE REVIEW

File: system/.LCKconfig.php~ [hunk 1]
```diff
@@ -0,0 +1 @@
+/Applications/AMPPS/www/2picrm/system/config.php
```
Comment (adam-buckley) [inline system/.LCKconfig.php~]: Looks like a random file was committed?

File: system/web.php [hunk 2]
```diff
@@ -344,16 +344,23 @@ function install() {
...
-		date_default_timezone_set($timezone);
+        // Set the timezone from Config
+        $timezone = Config::get('system.timezone');
+        if (empty($timezone)) {
+                $timezone = 'UTC';
+        }
+        date_default_timezone_set($timezone);
 		
+        //check config for 'gc_maxlifetime' for the session
+        $gc_maxlifetime = Config::get('system.gc_maxlifetime')[0];
```
Comment (adam-buckley) [inline system/web.php]: You shouldn't need the [0] part, the 'system.gc_maxlifetime' should be returning an integer...

File: system/config.php [hunk 3]
```diff
@@ -49,5 +49,7 @@
 	'sendmail' => 'SwiftMailerTransport'
 ));
 
+Config::set('system.gc_maxlifetime', 21600);
```
Comment (adam-buckley) [inline system/config.php]: This number doesn't fall into your lifetime bounds check in Web, 21600 seconds is 6 hours, but your lower bounds is a day

File: system/.LCKconfig.php~ (hunk 1 above)
Comment (iceaxeliehne) [inline system/.LCKconfig.php~]: removed

File: system/web.php (hunk 2 above)
Comment (iceaxeliehne) [inline system/web.php]: fixed the way I was setting the config. now works without square brackets

File: system/config.php (hunk 3 above)
Comment (iceaxeliehne) [inline system/config.php]: changed lower bounds to 1 hour

**Comment by iceaxeliehne** [comment]
requested changes have been made

**Pull request merged**